content/zh/docs/queue_resource_management.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "队列资源管理"
+
+date = 2024-12-30
+lastmod = 2024-12-30
+
+draft = false  # Is this a draft? true/false
+toc = true  # Show table of contents? true/false
+type = "docs"  # Do not modify.
+
+# Add menu entry to sidebar.
+[menu.docs]
+  parent = "features"
+  weight = 1
++++
+
+## 功能概述
+
+[队列](/zh/docs/queue)是Volcano的核心概念之一，用于支持多租户场景下的资源分配与任务调度。通过队列，用户可以实现多租资源分配、任务优先级控制、资源抢占与回收等功能，显著提升集群的资源利用率和任务调度效率。
+
+### 核心特性
+
+1. 灵活的资源配置
+
+   * 支持多维度资源配额控制(CPU、内存、GPU、NPU等)
+   * 提供三级资源配置机制:
+      * Capability: 队列资源使用上限
+      * Deserved: 资源应得量（在无其他队列提交作业时，该队列内作业所占资源量可超过Deserved值，当有多个队列提交作业且集群资源不够用时，超过Deserved值的资源量可以被其他队列回收）
+      * Guarantee: 资源预留量（预留资源只可被该队列所使用，其他队列无法使用）
+
+      > 建议: 
+      > 1. 进行三级资源配置时，需遵循: Guarantee <= Deserved <= Capability；
+      > 2. Guarantee可按需配置，在开启Capacity插件时需要配置Deserved值；
+      > 3. Deserved配置建议：在平级队列场景，所有队列的Deserved值总和等于集群资源总量；在层级队列场景，子队列的Deserved值总和等于父队列的Deserved值。
+
+   * 支持动态资源配额调整
+
+
+2.[层级队列](/zh/docs/hierarchical_queue)管理
+
+   * 支持多层级队列结构
+   * 提供父子队列间的资源继承与隔离
+   * 兼容Yarn式的资源管理模式，便于大数据工作负载迁移
+   * 支持跨层级队列的资源共享与回收
+
+3.智能资源调度
+
+   * 资源借用：允许队列使用其他队列的空闲资源
+   * 资源回收：当资源紧张时，优先回收超额使用的资源
+   * 资源抢占：确保高优先级任务的资源需求
+
+4.多租户隔离
+
+   * 严格的资源配额控制
+   * 基于优先级的资源分配
+   * 防止单个租户过度占用资源
+
+## 队列调度实现机制
+### 相关调度Actions
+Volcano中的队列调度涉及以下核心Action：
+
+1. Enqueue Action：控制作业进入队列的准入机制，根据队列的资源配额和当前使用情况决定是否允许新作业进入队列。
+
+2. Allocate Action：负责资源分配过程，确保分配符合队列配额限制，同时支持队列间的资源借用机制，提高资源利用率。
+
+3. Preempt Action：支持**队列内**资源抢占。高优先级作业可以抢占同队列内低优先级作业的资源，确保关键任务的及时执行。
+
+4. Reclaim Action：支持**队列间**的资源回收。当队列资源紧张时，触发资源回收机制。优先回收超出队列deserved值的资源，并结合队列/作业优先级选择合适的牺牲者。
+
+### 队列调度插件
+Volcano提供了两个核心的队列调度插件，它们都是基于Deserved值进行资源管理，区别在于Deserved值的设置方式：：
+#### 资源管理机制
+1. Deserved值
+   * 表示队列的资源应得量
+   * 当队列资源使用超过Deserved值时，超出部分可被其他队列回收
+   * 当其他队列资源空闲时，队列可以临时使用超过Deserved值的资源
+
+2. 资源回收机制
+   * 触发条件：集群资源紧张且某个队列需要使用资源时
+   * 回收对象：使用超过Deserved值的队列
+   * 回收过程：逐步回收超出部分，直到符合Deserved值
+
+#### Capacity Plugin
+Capacity Plugin支持通过显式配置Deserved值来设置队列资源应得量，如以下队列配置示例：
+```
+apiVersion: scheduling.volcano.sh/v1beta1
+kind: Queue
+metadata:
+  name: capacity-queue
+spec:
+  deserved:
+    cpu: "10"
+    memory: "20Gi"
+  capability:
+    cpu: "20"
+    memory: "40Gi"
+```
+Capacity Plugin通过精确的资源配置来进行配额控制，结合[层级队列](/zh/docs/hierarchical_queue)能实现更加精细的多租资源分配，也便于大数据工作负载迁移到Kubernetes集群上
+
+#### Proportion Plugin
+与Capacity Plugin不同的是，Proportion Plugin通过配置队列的Weight值来自动计算队列资源应得量，无需显式配置Deserved值，如以下队列配置示例：
+```
+apiVersion: scheduling.volcano.sh/v1beta1
+kind: Queue
+metadata:
+  name: proportion-queue
+spec:
+  weight: 1
+  capability:
+    cpu: "20"
+    memory: "40Gi"
+```
+Proportion插件可根据资源总量和队列数量自动调整队列的Deserved值，配置简单直观，适用于多租场景下各租户权重相同，对资源精确度要求不高的场景。如集群总资源为`total_resource`,
+此时有多个队列中同时提交了作业，则其中一个队列设置的Deserved值即为`(weight/total_weight)*total_resource`
+
```

Comment:
Add the notice that capacity and proportion are two different queue management way and cannot use together.